controllers/util/mw_util.go

```diff
@@ -74,34 +70,6 @@ func (mwu *MWUtil) BuildManifestWorkName(mwType string) string {
 	return ManifestWorkName(mwu.InstName, mwu.InstNamespace, mwType)
 }
 
-func (mwu *MWUtil) ManifestExistAndApplied(mwName, newPrimary string) (bool, error) {
```

Comment:
Why is this utility going away? I'd think this would be useful to check if VRG exists and is applied?

Reply:
No longer used anywhere, so to satisfy the linter, I removed it.